packages/swarmplot/index.d.ts

```diff
@@ -40,6 +56,8 @@ declare module '@nivo/swarmplot' {
         event: React.MouseEvent<any>
     ) => void
 
+    export type Layers<Datum> = ((props: LayerProps<Datum>) => JSX.Element) | string
```

Comment:
Can we make this consistent with other packages like `bar` or `line`. Change from string to a union of possible layers.

Reply:
Of course, ill get those changes pushed up tomorrow.